Description: Lilypond texinfo is missing direntries in some cases
 which breaks install-info; add them
Bug-Debian: https://bugs.debian.org/648689
Author: Julian Gilbey <jdg@debian.org>
Origin: vendor
Index: b/Documentation/en/contributor.texi
===================================================================
--- a/Documentation/en/contributor.texi	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/contributor.texi	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Contributor Guide: (lilypond-contributor).  Technical issues and policies for contributors.
+@end direntry
+
 @macro manualIntro
 This manual documents contributing to LilyPond version
 @version{}.  It discusses technical issues and policies that
Index: b/Documentation/cs/learning.tely
===================================================================
--- a/Documentation/cs/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/cs/learning.tely	2022-12-04 12:06:37.146961396 +0200
@@ -12,6 +12,11 @@
 @settitle Příručka k učení se
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* Příručka LilyPond k učení se: (lilypond-learning).  Začátek je zde.
+@end direntry
+
 @macro manualIntro
 Tento soubor poskytuje úvod pro LilyPond verze
 @version{}.
Index: b/Documentation/cs/web.texi
===================================================================
--- a/Documentation/cs/web.texi	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/cs/web.texi	2022-12-04 12:06:37.146961396 +0200
@@ -50,7 +50,6 @@ License''.
 @c expected to be found in lilypond/ subdirectory.
 @dircategory GNU LilyPond --- the music typesetter
 @direntry
-* Příručka LilyPond k učení se: (lilypond-learning).  Začátek je zde.
 * Slovníček: (music-glossary).               Překlad pojmů z angličtiny do dalších řečí.
 * Příručka pro zápis not v LilyPondu: (lilypond-notation).                           Příručka pro zápis not v LilyPondu.
 * Úryvky LilyPondu: (lilypond-snippets).         Mnoho triků, rad a příkladů.
Index: b/Documentation/de/essay.tely
===================================================================
--- a/Documentation/de/essay.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/de/essay.tely	2022-12-04 12:06:37.146961396 +0200
@@ -12,6 +12,11 @@
 @documentlanguage de
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond automatischen Musiksatz: (lilypond-essay).  Automatischen Musiksatz von LilyPond.
+@end direntry
+
 @c Translators: Till Paala
 
 @macro manualIntro
Index: b/Documentation/de/extending.tely
===================================================================
--- a/Documentation/de/extending.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/de/extending.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage de
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Erweiterung: (lilypond-extending).  Wie man die Funktionalität von LilyPond erweitern kann.
+@end direntry
+
 @c Translators: Till Paala
 
 @macro manualIntro
Index: b/Documentation/de/learning.tely
===================================================================
--- a/Documentation/de/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/de/learning.tely	2022-12-04 12:06:37.146961396 +0200
@@ -12,6 +12,11 @@
 @settitle LilyPond Handbuch zum Lernen
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Handbuch zum Lernen: (lilypond-learning).  Hier beginnen.
+@end direntry
+
 @macro manualIntro
 Diese Datei stellt eine Einleitung für LilyPond Version
 @version{} zur Verfügung.
Index: b/Documentation/de/notation.tely
===================================================================
--- a/Documentation/de/notation.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/de/notation.tely	2022-12-04 12:06:37.146961396 +0200
@@ -11,6 +11,11 @@
 @setfilename lilypond.info
 @settitle LilyPond Benutzerhandbuch
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Notationshandbuch: (lilypond-notation).                           LilyPond Notationshandbuch.
+@end direntry
+
 @c Translators: Till Paala
 
 @afourpaper
Index: b/Documentation/de/usage.tely
===================================================================
--- a/Documentation/de/usage.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/de/usage.tely	2022-12-04 12:06:37.146961396 +0200
@@ -12,7 +12,17 @@
 @settitle GNU LilyPond Programmbenutzung
 @afourpaper
 
-
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Application Usage: (lilypond-usage). Programme installieren und benutzen.
+* lilypond: (lilypond-usage)LilyPond starten.  Das LilyPond-Programm aufrufen.
+* abc2ly: (lilypond-usage)abc2ly starten.      ABC-Notation importieren.
+* convert-ly: (lilypond-usage)Dateien mit convert-ly aktualisieren. Ältere LilyPond-Versionen
+* etf2ly: (lilypond-usage)etf2ly starten.      Finale-Dateien importieren.
+* lilypond-book: (lilypond-usage)LilyPond-book. Text und Musik integrieren.
+* midi2ly: (lilypond-usage)midi2ly starten.    MIDI importieren.
+* musicxml2ly: (lilypond-usage)musicxml2ly starten.  MusicXML-Dateien importieren.
+@end direntry
 
 @macro manualIntro
 Diese Datei erklärt, wie man die Programme, die mit LilyPond
Index: b/Documentation/es/essay.tely
===================================================================
--- a/Documentation/es/essay.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/es/essay.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage es
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* Las funciones de grabado musical automatizado: (lilypond-essay).  Las funciones de grabado musical automatizado dentro de LilyPond.
+@end direntry
+
 @c translator: Francisco Vila
 
 @macro manualIntro
Index: b/Documentation/es/extending.tely
===================================================================
--- a/Documentation/es/extending.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/es/extending.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage es
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Extensión: (lilypond-extending).  Extender las funcionalidades de LilyPond.
+@end direntry
+
 @c translator: Francisco Vila
 
 @macro manualIntro
Index: b/Documentation/es/learning.tely
===================================================================
--- a/Documentation/es/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/es/learning.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage es
 @afourpaper
 
+@dircategory GNU LilyPond: el tipografiador de música
+@direntry
+* Manual de aprendizaje de LilyPond: (lilypond-learning).  Comience por aquí.
+@end direntry
+
 @macro manualIntro
 Este archivo ofrece una introducción al programa LilyPond versión
 @version{}.
Index: b/Documentation/es/notation.tely
===================================================================
--- a/Documentation/es/notation.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/es/notation.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage es
 @afourpaper
 
+@dircategory GNU LilyPond: el tipografiador de música
+@direntry
+* LilyPond: (lilypond-notation).                      Referencia de la notación con LilyPond.
+@end direntry
+
 @macro manualIntro
 Este manual ofrece una referencia para toda la notación musical que se
 puede producir con LilyPond versión @version{}.  Da por sentado que el
Index: b/Documentation/en/essay.tely
===================================================================
--- a/Documentation/en/essay.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/essay.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Engraving Functions: (lilypond-essay).  Automatic engraving functions within LilyPond.
+@end direntry
+
 @macro manualIntro
 This essay discusses automatic music engraving functions within
 LilyPond version @version{}.
Index: b/Documentation/es/usage.tely
===================================================================
--- a/Documentation/es/usage.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/es/usage.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,18 @@
 @documentlanguage es
 @afourpaper
 
+@dircategory GNU LilyPond: el tipografiador de música
+@direntry
+* Utilización del programa: (lilypond-usage). Instalación y ejecución de las aplicaciones.
+* lilypond: (lilypond-usage)Ejecutar LilyPond.      Invocar el programa LilyPond.
+* abc2ly: (lilypond-usage)Invocación de abc2ly.      Importación de ABC.
+* convert-ly: (lilypond-usage)Actualización de archivos mediante convert-ly. Versiones antiguas de LilyPond.
+* etf2ly: (lilypond-usage)Invocar etf2ly.      Importación de Finale.
+* lilypond-book: (lilypond-usage)LilyPond-book. Intergrar texto y música.
+* midi2ly: (lilypond-usage)Invocar midi2ly.    Importación de MIDI.
+* musicxml2ly: (lilypond-usage)Invocar musicxml2ly.  Importación de MusicXML.
+@end direntry
+
 @c Translators: Francisco Vila
 
 @macro manualIntro
Index: b/Documentation/es/web.texi
===================================================================
--- a/Documentation/es/web.texi	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/es/web.texi	2022-12-04 12:06:37.146961396 +0200
@@ -53,20 +53,10 @@ Free Documentation License''.
 @c expected to be found in lilypond/ subdirectory.
 @dircategory GNU LilyPond: el tipografiador de música
 @direntry
-* Manual de aprendizaje de LilyPond: (lilypond-learning).  Comience por aquí.
 * Glosario musical: (music-glossary).                 Para usuarios de habla no inglesa.
-* LilyPond: (lilypond-notation).                      Referencia de la notación con LilyPond.
 * Fragmentos de código de LilyPond: (lilypond-snippets).         Consejos, trucos y ejemplos cortos.
 * Referencia de funcionamiento interno: (lilypond-internals).  Definiciones para la construcción de trucos.
-* Utilización del programa: (lilypond-usage). Instalación y ejecución de las aplicaciones.
 * Página web: (lilypond-web).    Vista previa de la nueva página.
-* lilypond: (lilypond-usage)Ejecutar LilyPond.      Invocar el programa LilyPond.
-* abc2ly: (lilypond-usage)Invocación de abc2ly.      Importación de ABC.
-* convert-ly: (lilypond-usage)Actualización de archivos mediante convert-ly. Versiones antiguas de LilyPond.
-* etf2ly: (lilypond-usage)Invocar etf2ly.      Importación de Finale.
-* lilypond-book: (lilypond-usage)LilyPond-book. Intergrar texto y música.
-* midi2ly: (lilypond-usage)Invocar midi2ly.    Importación de MIDI.
-* musicxml2ly: (lilypond-usage)Invocar musicxml2ly.  Importación de MusicXML.
 @end direntry
 
 
Index: b/Documentation/en/extending.tely
===================================================================
--- a/Documentation/en/extending.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/extending.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Extending: (lilypond-extending).  How to extend LilyPond's functionality.
+@end direntry
+
 @macro manualIntro
 This file explains how to extend the functionality of
 LilyPond version @version{}.
Index: b/Documentation/fr/essay.tely
===================================================================
--- a/Documentation/fr/essay.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/fr/essay.tely	2022-12-04 12:06:37.146961396 +0200
@@ -16,6 +16,11 @@
 @documentlanguage fr
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* La gravure musicale automatisée: (lilypond-essay).  La gravure musicale automatisée de LilyPond.
+@end direntry
+
 @macro manualIntro
 Cet essai traite de la gravure de la musique et de la notation assistée
 par ordinateur à l'aide de LilyPond version @version{}.
Index: b/Documentation/fr/learning.tely
===================================================================
--- a/Documentation/fr/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/fr/learning.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 
 @afourpaper
 
+@dircategory GNU LilyPond -- le typographe de musique
+@direntry
+* Manuel d'initiation à LilyPond: (lilypond-learning).       Les premiers pas.
+@end direntry
+
 @macro manualIntro
 Ce document constitue le manuel d'initiation à GNU LilyPond
 @version{}.
Index: b/Documentation/fr/notation.tely
===================================================================
--- a/Documentation/fr/notation.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/fr/notation.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage fr
 @afourpaper
 
+@dircategory GNU LilyPond -- le typographe de musique
+@direntry
+* Manuel de notation: (lilypond-notation).                   Gravure de musique avec LilyPond.
+@end direntry
+
 @macro manualIntro
 Ce document constitue le manuel de notation de GNU LilyPond
 @version{}.  Sa lecture requiert une familiarité avec le contenu
Index: b/Documentation/fr/usage.tely
===================================================================
--- a/Documentation/fr/usage.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/fr/usage.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,17 @@
 @documentlanguage fr
 @afourpaper
 
+@dircategory GNU LilyPond -- le typographe de musique
+@direntry
+* Manuel d'utilisation: (lilypond-usage).                    Installation et exécution des programmes.
+* abc2ly: (lilypond-usage) Lancement de abc2ly.              Import de fichier ABC.
+* convert-ly: (lilypond-usage) Mise à jour de fichiers.      Pour d'anciens fichiers LilyPond.
+* etf2ly: (lilypond-usage) Lancement de etf2ly.              Import de fichier Finale.
+* lilypond-book: (lilypond-usage) LilyPond-book.             Association musique-texte.
+* midi2ly: (lilypond-usage) Lancement de midi2ly.            Import de fichier MIDI.
+* musicxml2ly: (lilypond-usage) Lancement de musicxml2ly.    Import de fichier MusicXML.
+@end direntry
+
 @c Translators: John Mandereau, Jean-Charles Malahieude
 
 @macro manualIntro
Index: b/Documentation/hu/learning.tely
===================================================================
--- a/Documentation/hu/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/hu/learning.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage hu
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* Tankönyv: (lilypond-learning).  Itt kezdd az olvasást.
+@end direntry
+
 @c Translators: Harmath Dénes
 
 @macro manualIntro
Index: b/Documentation/hu/web.texi
===================================================================
--- a/Documentation/hu/web.texi	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/hu/web.texi	2022-12-04 12:06:37.146961396 +0200
@@ -51,20 +51,11 @@ License''.
 @c expected to be found in lilypond/ subdirectory.
 @dircategory GNU LilyPond --- the music typesetter
 @direntry
-* Tankönyv: (lilypond-learning).  Itt kezdd az olvasást.
 * Fogalomtár: (music-glossary).               Többnyelvű szószedet.
 * Kottaírás: (lilypond-notation).                           A kottaírás kézikönyve.
 * Kódrészletek: (lilypond-snippets).         Tippek, trükkök, példák.
 * Belső működés: (lilypond-internals).  A belső működés referenciája.
-* Használat: (lilypond-usage). Az alkalmazás használata.
 * Web: (lilypond-web).    A weboldal.
-* lilypond: (lilypond-usage)A lilypond futtatása.      Kották szedése.
-* abc2ly: (lilypond-usage)Az abc2ly futtatása.      ABC fájlok importálása.
-* convert-ly: (lilypond-usage)A convert-ly futtatása. Konvertálás régebbi verziókról.
-* etf2ly: (lilypond-usage)Az etf2ly futtatása.      Finale fájlok importálása.
-* lilypond-book: (lilypond-usage)A lilypond-book futtatása. Szöveg és zene integrálása.
-* midi2ly: (lilypond-usage)A midi2ly futtatása.    MIDI fájlok importálása.
-* musicxml2ly: (lilypond-usage)A musicxml2ly futtatása.  MusicXML fájlok importálása.
 @end direntry
 
 
Index: b/Documentation/it/learning.tely
===================================================================
--- a/Documentation/it/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/it/learning.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage it
 @afourpaper
 
+@dircategory GNU LilyPond --- il compositore tipografico di musica
+@direntry
+* Manuale di Apprendimento di LilyPond: (lilypond-learning).    Comincia da qui.
+@end direntry
+
 @macro manualIntro
 Questo file fornisce un'introduzione alla versione di LilyPond
 @version{}.
Index: b/Documentation/it/usage.tely
===================================================================
--- a/Documentation/it/usage.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/it/usage.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,18 @@
 @documentlanguage it
 @afourpaper
 
+@dircategory GNU LilyPond --- il compositore tipografico di musica
+@direntry
+* Uso del Programma: (lilypond-usage).                    Istallazione e esecuzione delle applicazioni.
+* lilypond: (lilypond-usage)Eseguire lilypond.                  Invocare il programma LilyPond.
+* abc2ly: (lilypond-usage)Invocare abc2ly.                      Importare ABC.
+* convert-ly: (lilypond-usage)Aggiornare i file con convert-ly. Versioni precedenti di LilyPond.
+* etf2ly: (lilypond-usage)Invocare etf2ly.                      Importare Finale.
+* lilypond-book: (lilypond-usage)lilypond-book.                 Integrare testo e musica.
+* midi2ly: (lilypond-usage)Invocare midi2ly.                    Importare MIDI.
+* musicxml2ly: (lilypond-usage)Invocare musicxml2ly.            Importare MusicXML.
+@end direntry
+
 @c Translators: Federico Bruni
 @c Translation checkers: Luca Rossetto Casel
 
Index: b/Documentation/ja/learning.tely
===================================================================
--- a/Documentation/ja/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/ja/learning.tely	2022-12-04 12:06:37.146961396 +0200
@@ -13,6 +13,11 @@
 @documentlanguage ja
 @afourpaper
 
+@dircategory GNU LilyPond --- 楽譜譜刻プログラム
+@direntry
+* LilyPond 学習マニュアル: (lilypond-learning).        ここから始めてください。
+@end direntry
+
 @macro manualIntro
 このマニュアルは LilyPond バージョン @version{} の導入部です。
 @end macro
Index: b/Documentation/ja/notation.tely
===================================================================
--- a/Documentation/ja/notation.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/ja/notation.tely	2022-12-04 12:06:37.150961212 +0200
@@ -13,6 +13,11 @@
 @documentlanguage ja
 @afourpaper
 
+@dircategory GNU LilyPond --- 楽譜譜刻プログラム
+@direntry
+* LilyPond: (lilypond-notation).                       LilyPond 記譜法リファレンス。
+@end direntry
+
 @macro manualIntro
 このマニュアルは LilyPond バージョン @version{} で提供される@c
 すべての記譜法についての参照を提供します。@c
Index: b/Documentation/ja/usage.tely
===================================================================
--- a/Documentation/ja/usage.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/ja/usage.tely	2022-12-04 12:06:37.150961212 +0200
@@ -13,6 +13,19 @@
 @documentlanguage ja
 @afourpaper
 
+@dircategory GNU LilyPond --- 楽譜譜刻プログラム
+@direntry
+* LilyPond アプリケーション使用方法: (lilypond-usage). アプリケーションを@c
+インスールして、実行する方法。
+* lilypond: (lilypond-usage)lilypond を実行する.       LilyPond を呼び出す。
+* abc2ly: (lilypond-usage)abc2ly を呼び出す.           ABC をインポートする。
+* convert-ly: (lilypond-usage)convert-ly でファイルを更新する. 古い LilyPond バージョン。
+* etf2ly: (lilypond-usage)etf2ly を呼び出す.           Finale をインポートする。
+* lilypond-book: (lilypond-usage)lilypond-book.        文章と楽譜を統合する。
+* midi2ly: (lilypond-usage)midi2ly を呼び出す.         MIDI をインポートする。
+* musicxml2ly: (lilypond-usage)musicxml2ly を呼び出す. MusicXML をインポートする。
+@end direntry
+
 @macro manualIntro
 このマニュアルは LilyPond バージョン @version{} で配布される@c
 プログラムの実行方法について説明します。@c
Index: b/Documentation/en/learning.tely
===================================================================
--- a/Documentation/en/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/learning.tely	2022-12-04 12:06:37.150961212 +0200
@@ -13,6 +13,11 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Learning Manual: (lilypond-learning).  Start here.
+@end direntry
+
 @macro manualIntro
 This file provides an introduction to LilyPond version
 @version{}.
Index: b/Documentation/en/music-glossary.tely
===================================================================
--- a/Documentation/en/music-glossary.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/music-glossary.tely	2022-12-04 12:06:37.150961212 +0200
@@ -6,6 +6,11 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* Music Glossary: (music-glossary).               For non-English users.
+@end direntry
+
 @macro manualIntro
 This glossary provides definitions and translations of musical
 terms used in the documentation manuals for LilyPond version
Index: b/Documentation/nl/learning.tely
===================================================================
--- a/Documentation/nl/learning.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/nl/learning.tely	2022-12-04 12:06:37.150961212 +0200
@@ -19,6 +19,11 @@
 @documentlanguage nl
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Learning Manual: (lilypond-learning).  Start here.
+@end direntry
+
 @macro manualIntro
 This file provides an introduction to LilyPond version
 @version{}.
Index: b/Documentation/en/notation.tely
===================================================================
--- a/Documentation/en/notation.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/notation.tely	2022-12-04 12:06:37.150961212 +0200
@@ -13,6 +13,11 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond: (lilypond-notation).                           LilyPond Notation Reference.
+@end direntry
+
 @macro manualIntro
 This manual provides a reference for all notation that can be
 produced with LilyPond version @version{}.  It assumes that
Index: b/Documentation/en/snippets.tely
===================================================================
--- a/Documentation/en/snippets.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/snippets.tely	2022-12-04 12:06:37.150961212 +0200
@@ -6,6 +6,11 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Snippets: (lilypond-snippets).         Short tricks, tips, and examples.
+@end direntry
+
 @macro manualIntro
 This document shows a selected set of LilyPond snippets from the
 @uref{https://lsr@/.di@/.unimi@/.it,LilyPond Snippet Repository}
Index: b/Documentation/en/usage.tely
===================================================================
--- a/Documentation/en/usage.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/usage.tely	2022-12-04 12:06:37.150961212 +0200
@@ -13,6 +13,18 @@
 @documentlanguage en
 @afourpaper
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Application Usage: (lilypond-usage). Installing and running applications.
+* lilypond: (lilypond-usage)Running lilypond.      Invoking the LilyPond program.
+* abc2ly: (lilypond-usage)Invoking abc2ly.      Importing ABC.
+* convert-ly: (lilypond-usage)Updating files with convert-ly. Older LilyPond versions.
+* etf2ly: (lilypond-usage)Invoking etf2ly.      Importing Finale.
+* lilypond-book: (lilypond-usage)lilypond-book. Integrating text and music.
+* midi2ly: (lilypond-usage)Invoking midi2ly.    Importing MIDI.
+* musicxml2ly: (lilypond-usage)Invoking musicxml2ly.  Importing MusicXML.
+@end direntry
+
 @macro manualIntro
 This file explains how to execute the programs distributed with
 LilyPond version @version{}.  In addition, it suggests some
Index: b/Documentation/en/changes.tely
===================================================================
--- a/Documentation/en/changes.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/changes.tely	2022-12-04 12:06:37.150961212 +0200
@@ -53,6 +53,10 @@ run @command{convert-ly} directly from F
 environments with LilyPond support may provide a way to run
 @command{convert-ly} graphically.}
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* LilyPond Changes: (lilypond-changes).   Changes to recent versions.
+@end direntry
 
 @node Major changes in LilyPond
 @unnumbered Major changes in LilyPond
Index: b/Documentation/hu/usage.tely
===================================================================
--- a/Documentation/hu/usage.tely	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/hu/usage.tely	2022-12-04 12:06:37.150961212 +0200
@@ -15,6 +15,19 @@
 
 @c translator: Harmath Dénes
 
+@dircategory GNU LilyPond --- the music typesetter
+@direntry
+* Használat: (lilypond-usage). Az alkalmazás használata.
+* lilypond: (lilypond-usage)A lilypond futtatása.      Kották szedése.
+* abc2ly: (lilypond-usage)Az abc2ly futtatása.      ABC fájlok importálása.
+* convert-ly: (lilypond-usage)A convert-ly futtatása. Konvertálás régebbi verziókról.
+* etf2ly: (lilypond-usage)Az etf2ly futtatása.      Finale fájlok importálása.
+* lilypond-book: (lilypond-usage)A lilypond-book futtatása. Szöveg és zene integrálása.
+* midi2ly: (lilypond-usage)A midi2ly futtatása.    MIDI fájlok importálása.
+* musicxml2ly: (lilypond-usage)A musicxml2ly futtatása.  MusicXML fájlok importálása.
+@end direntry
+
+
 @macro manualIntro
 Ez a dokumentáció ismerteti, hogyan kell a LilyPond @version{} verziójához
 tartozó programokat futtatni, valamint tanácsokat ad azok hatékony
Index: b/Documentation/en/web.texi
===================================================================
--- a/Documentation/en/web.texi	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/en/web.texi	2022-12-04 12:06:37.150961212 +0200
@@ -1,3 +1,4 @@
+
 \input texinfo @c -*- coding: utf-8; mode: texinfo; -*-
 @ignore
     Translation of GIT committish: FILL-IN-HEAD-COMMITTISH
Index: b/Documentation/de/web.texi
===================================================================
--- a/Documentation/de/web.texi	2022-11-27 14:28:56.000000000 +0200
+++ b/Documentation/de/web.texi	2022-12-04 12:06:37.150961212 +0200
@@ -52,16 +52,8 @@ License''.
 @c expected to be found in lilypond/ subdirectory.
 @dircategory LilyPond -- das Notensatzsystem
 @direntry
-* Neuerungen: (lilypond-changes).       Neuerungen seit Version 2.20.
-* Beitragen: (lilypond-contributor).    Zu LilyPond beitragen.
-* Aufsatz: (lilypond-essay).            Die Philosophie hinter LilyPond.
-* Extending: (lilypond-extending).      Scheme in LilyPond benutzen.
 * Interna: (lilypond-internals).        Referenz für Anpassungen.
-* Lernen: (lilypond-learning).          Einführung.  Hier starten.
-* Glossar: (music-glossary).            Begriffsübersetzungen aus dem Englischen.
-* Notation: (lilypond-notation).        Notationshandbuch.
 * Schnipsel: (lilypond-snippets).       Tricks, Tipps und Beispiele.
-* Benutzung: (lilypond-usage).          LilyPond installieren und benutzen.
 * Webseite: (lilypond-web).             Info-Version von lilypond.org.
 @end direntry
 
